templates/config.cli.erb

```diff
@@ -83,3 +83,38 @@ catch
 /subsystem=keycloak-server/spi=userCache/provider=default/:remove
 /subsystem=keycloak-server/spi=userCache/provider=default/:add(enabled=<%= scope['keycloak::user_cache']%>)
 end-try
+<%- if scope['keycloak::operating_mode'] == 'clustered' && scope['keycloak::enable_jdbc_ping'] -%>
+/subsystem=jgroups/stack=tcp/protocol=MPING: remove()
+/socket-binding-group=standard-sockets/socket-binding=jgroups-mping:remove()
+/subsystem=jgroups/stack=tcp/protocol=MERGE3: remove()
+/subsystem=jgroups/stack=tcp/protocol=FD_SOCK: remove()
+/subsystem=jgroups/stack=tcp/protocol=FD_ALL: remove()
+/subsystem=jgroups/stack=tcp/protocol=VERIFY_SUSPECT: remove()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.NAKACK2: remove()
+/subsystem=jgroups/stack=tcp/protocol=UNICAST3: remove()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.STABLE: remove()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.GMS: remove()
+/subsystem=jgroups/stack=tcp/protocol=MFC: remove()
+/subsystem=jgroups/stack=tcp/protocol=FRAG3: remove()
+<%- if scope['keycloak::datasource_driver'] == 'postgresql' -%>
+/subsystem=jgroups/stack=tcp/protocol=JDBC_PING: add(data-source="KeycloakDS", properties=[initialize_sql="CREATE TABLE IF NOT EXISTS JGROUPSPING (own_addr varchar(200) NOT NULL, creation_timestamp timestamp, cluster_name varchar(200) NOT NULL, ping_data bytea, constraint PK_JGROUPSPING PRIMARY KEY (own_addr, cluster_name))"])
+<%- end -%>
+<%- if scope['keycloak::datasource_driver'] == 'mysql' -%>
+/subsystem=jgroups/stack=tcp/protocol=JDBC_PING: add(data-source="KeycloakDS", properties=[initialize_sql="CREATE TABLE IF NOT EXISTS JGROUPSPING (own_addr varchar(200) NOT NULL, cluster_name varchar(200) NOT NULL, updated TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, ping_data varbinary(5000) DEFAULT NULL, PRIMARY KEY (own_addr, cluster_name)) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin"])
+<%- end -%>
+/subsystem=jgroups/stack=tcp/protocol=MERGE3: add()
+/subsystem=jgroups/stack=tcp/protocol=FD_SOCK: add()
+/subsystem=jgroups/stack=tcp/protocol=FD_ALL: add()
+/subsystem=jgroups/stack=tcp/protocol=VERIFY_SUSPECT: add()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.NAKACK2: add()
+/subsystem=jgroups/stack=tcp/protocol=UNICAST3: add()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.STABLE: add()
+/subsystem=jgroups/stack=tcp/protocol=pbcast.GMS: add(properties=[join_timeout=30000, print_local_addr=true, print_physical_addrs=true])
+/subsystem=jgroups/stack=tcp/protocol=MFC: add()
+/subsystem=jgroups/stack=tcp/protocol=FRAG2: add()
+/subsystem=jgroups/channel=ee:write-attribute(name=stack, value="tcp")
+/subsystem=jgroups/stack=udp: remove()
+/socket-binding-group=standard-sockets/socket-binding=jgroups-udp:remove()
+/interface=private:write-attribute(name=inet-address, value=${jboss.bind.address.private:<%= scope['::ipaddress'] %>})
```

Comment:
This and the public value should either be `0.0.0.0` or a parameter to define the addresses that defaults to `$facts['networking']['ip']`.